File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Arrays.sort(nums);
        int count = 0;
        for(int i = 1;i<nums.length;i++){
            if(nums[i] == nums[i-1]){
                count++;
            }
        }
        if(count>0){
            return true;
        }
        else{
            return false;
        }
    }
}

File: Data Structures & Algorithms/subsets-ii/submission-0.py
class Solution:
    def sub(self,nums,result,temp,i):
        if i == len(nums):
            result.append(temp[:])
            return
        
        temp.append(nums[i])
        self.sub(nums,result,temp,i+1)
        temp.pop()
        while i+1<len(nums) and nums[i] == nums[i+1]:
            i+=1
        self.sub(nums,result,temp,i+1)
        return result
        
    def subsetsWithDup(self, nums: List[int]) -> List[List[int]]:
        nums.sort()
        return self.sub(nums,[],[],0)
        